cocktails.web/.env

```
@@ -8,11 +8,13 @@ VITE_AUTH0_DOMAIN=login.cezzis.com
 VITE_AUTH0_CLIENT_ID=HZXyiZxjHgkQqeb6UJxmikCGYpSz5iPb
 VITE_AUTH0_REDIRECT_URI=https://localhost:4001/iam/auth/redirect/
 VITE_AUTH0_COCKTAILS_API_AUDIENCE=https://cezzis-cocktails-api
+VITE_AUTH0_ACCOUNTS_API_AUDIENCE=https://cezzis-accounts-api
 
 VITE_COCKTAILS_API_URL=http://localhost:7177
 VITE_COCKTAILS_IMAGE_URL=http://localhost:7177/api/v1/images
 VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY=383hudiudhUJK984jdus7HDY
 
 VITE_AISEARCH_API_URL=http://localhost:8010
+VITE_ACCOUNTS_API_URL=http://localhost:8016
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ACCOUNTS_API_URL key should go before the VITE_AISEARCH_API_URL key

(UnorderedKey)